website/database.py

```diff
@@ -268,6 +268,19 @@ def delete_program_by_id(self, id):
         """Delete a program by id."""
         PROGRAMS.delete({"id": id})
 
+    def student_adventure_by_id(self, id):
+        # Fetch a student adventure with id formatted as studentID-adventureName-level
+        return STUDENT_ADVENTURES.get({"id": id})
+
+    def update_student_adventure(self, id, ticked):
+        # Swap the ticked value when a request is sent
+        return STUDENT_ADVENTURES.update({"id": id}, {"ticked": not ticked})
+
+    def store_student_adventure(self, adventure):
```

Comment:
Adventure is a name that is so used across the entire codebase and is often used to mean an adventure that will be displayed in the index, can you use a more descriptive name, maybe `student_adventure`?